src/NLog.Redis/RedisConnectionManager.cs

```diff
@@ -1,21 +1,32 @@
 ﻿using System;
 using StackExchange.Redis;
+using System.Collections.Generic;
 
 namespace NLog.Targets
 {
     internal class RedisConnectionManager : IDisposable
     {
         private ConnectionMultiplexer _connectionMultiplexer;
 
-        private readonly string _host;
+        private readonly List<string> _hosts;
         private readonly int _port;

```

Comment:
It seems that this is never used. Should it be removed?

Reply:
What do you think should be removed ?

Reply:
Oh, it didn't mark that row. I meant that private readonly int _port. At least it seemd to be unnecessary and my Visual Studio was giving a warning from it.

Reply:
You are absolutely right, I didn't see it before, correction on the way ! (And i'm happy to see people using this code by the way !)